plugins/kubernetes/app/models/kubernetes/resource_template.rb

```diff
@@ -210,6 +211,12 @@ def set_env
       end
     end
 
+    def set_image_pull_secrets
+      return unless secrets = ENV['KUBERNETES_IMAGE_PULL_SECRETS']
```

Comment:
not sure if it will be solved with 1.3 so I'd rather wait with that until I can verify it works